core/templates/pages/partnerships-page/partnerships-page.component.ts

```diff
@@ -67,6 +68,19 @@ export class PartnershipsPageComponent implements OnInit, OnDestroy {
     this.pageTitleService.setDocumentTitle(translatedTitle);
   }
 
+  getFormLink(): string {
+    let userLang = this.translateService.currentLang;
+
+    if (userLang === 'en' || userLang === 'pcm' || userLang === 'kab') {
+      this.formLink = 'https://forms.gle/Y71U8FdhQwZpicJj8';
+    } else if (userLang === 'pt-br') {
```

Comment:
Can you combine the pt-br and else clause into a single branch, and in that branch -- define a new variable interpolatedLanguage that copies userLang generally (but changes pt-br to pt) and then use that variable in the link? Then you can avoid repeating the long link twice.